demo/mmpose_cfg/hrnet_w48_coco_wholebody_384x288_dark_plus.py

```diff
@@ -0,0 +1,160 @@
+load_from = 'https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w48_coco_384x288_dark-741844ba_20200812.pth'  # noqa: E501
```

Comment:
Where do we need `load_from`? It seems that `pose_checkpoint` in [demo/pymafx_estimate_smplx.py](https://github.com/open-mmlab/mmhuman3d/pull/306/files#diff-d2c32733320f01f6c05ad58cb4e2d9324ea5bc7ee76e829c5a2b171fd680f0b2R467) has already specified the checkpoint.

Reply:
I deleted it.